cmd/asset-syncer/postgresql_utils.go

```diff
@@ -247,3 +247,14 @@ func (m *postgresAssetManager) insertFiles(chartFilesID string, files models.Cha
 	}
 	return err
 }
+
+// InvalidateCache for postgresql deletes and re-writes the schema
+func (m *postgresAssetManager) InvalidateCache() error {
+	tables := strings.Join([]string{dbutils.RepositoryTable, dbutils.ChartTable, dbutils.ChartFilesTable}, ",")
+	_, err := m.DB.Exec(fmt.Sprintf("DROP TABLE IF EXISTS %s CASCADE", tables))
+	if err != nil {
+		return err
+	}
+
+	return m.initTables()
```

Comment:
this is not really necessary since it will be executed with syncing but I am okay if you want to leave it to leave the tables ready

Reply:
Yeah, the reason for having it here is so that it is sync'd with the correct code (the new, rather than from the still running older pods). We will probably need a transaction here to avoid race conditions with the existing pods too.